docs/source/schema/creating-directives.mdx

```diff
@@ -1,56 +1,244 @@
 ---
-title: Implementing directives
-description: Implementing custom directives to transform schema types, fields, and arguments
+title: Creating schema directives
+description: Apply custom logic to GraphQL types, fields, and arguments
 ---
 
-Before learning how to implement schema directives, [this section](/schema/directives/) will provide the necessary background on schema directives and their use.
+import {ExpansionPanel} from 'gatsby-theme-apollo-docs';
 
-## Implementing schema directives
+> Before you create a custom schema directive, [learn the basics about directives](./directives).
 
-Since the GraphQL specification does not discuss any specific implementation strategy for directives, it's up to each GraphQL server framework to expose an API for implementing new directives.
+Your schema can define custom directives that can then decorate _other_ parts of your schema. Apollo Server can execute custom logic whenever it encounters a particular directive in your schema, such as by transforming data returned by resolvers.
```

Comment:
I think we should be clear that this processing is happening at startup when your schema is loaded, not at request time (but one of the things you can do is replace a resolver to add request-time behavior).